app/scripts/lib/errors.js

```diff
@@ -100,15 +100,25 @@ define([
      * @param {String} [context]
      */
     toError: function (type, context) {
-      var message = this.toMessage(type);
-      if (! message) {
```

Comment:
Should we still have a check to see if message exists? I think we should. @shane-tomlinson